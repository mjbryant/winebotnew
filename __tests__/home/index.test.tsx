import React from "react";
import TestRenderer from "react-test-renderer";
import Home from "../../src/pages/home/index";

describe("Home", () => {
  it("should render without throwing an error", () => {
    const testRenderer = TestRenderer.create(<Home />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
