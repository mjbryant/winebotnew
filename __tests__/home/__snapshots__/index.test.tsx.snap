// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render without throwing an error 1`] = `
[
  <div>
    <a
      className="menuLink"
      href="/home"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      Home
    </a>
    <a
      className="menuLink"
      href="/wines"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      Wines
    </a>
  </div>,
  <main
    className="main"
  >
    <div
      className="thing"
    >
      <div
        className="item"
        onClick={[Function]}
      >
        View wines
      </div>
      <div
        className="item"
        onClick={[Function]}
      >
        Blind taste
      </div>
    </div>
  </main>,
]
`;
